contracts/TokenManager.sol

```diff
@@ -120,6 +203,7 @@ contract TokenManager is System, IApplication {
   }
 
   function approveBind(address contractAddr, string memory bep2Symbol) payable public returns (bool) {
+    require(!mirrorPendingRecord[contractAddr], "the bep20 token is in mirror pending status");
     bytes32 bep2TokenSymbol = bep2TokenSymbolConvert(bep2Symbol);
```

Comment:
Please handle if mirror succeed.

Reply:
Good idea

Reply:
If user already mirror success, looks like apporveBind will success so far, won't it confict?

Reply:
`apporveBind ` will return error code for already bound